### ISSUE TITLE
Cannot find type definition file for 'node'

### ISSUE THREAD

**Issue by xvilo** [Mixbook/mixbook_http_client.ts#15]
Hi,

I have installed the client, and have added it with:

```ts
import {BrowserHttpClient} from "mixbook-http-client/build/browser";

class Something {
// asdf
}
```

When trying to compile this, I get the follow error:
`TypeScript error: node_modules/mixbook-http-client/build/httpclient.d.ts(1,23): Error TS2688: Cannot find type definition file for 'node'.`

package.json dependencies:

```
    "tsify": "^4.0.0",
    "typescript": "^2.2.2",
    "mixbook-http-client": "^0.3.4"
```

**Comment by astashov** [comment]
Thanks for reporting!

As a quick fix, you can add:

```json
"devDependencies": {
  "@types/node": "^10.1.3"
}
```

to your package.json. It should fix the issue.

But I see how that can be annoying if you only gonna use the client from the browser - why to add Node typings for a browser-only app?

I'll think about it more - how to avoid adding Node typings for browser-only usages of the library.

**Comment by astashov** [comment]
Okay, newly published version (0.4.0) should not require that anymore. I'm closing the issue, please reopen if it still happens for you.

Thanks!
